examples/test/legacy/test_i_v3_default_cmm.py

```diff
@@ -0,0 +1,20 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
```

Comment:
Either put the V3 CMM in this file, or in another file in this folder, then add a test for it in this file.
We don't want to put a copy-paste of the Default CMM in any src/ directory. We're only copy-pasting it for testing, so let's put it in a test directory